pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  intl: ^0.19.0
```

Comment:
Having the latest intl version can have impact on the compatibility with other projects.

I would like to keep this package as pure as possible without any external dependency aside from flutter.

@Gorter-dev What do you think about adding dependencies like this?